ssl/statem/extensions.c

```diff
@@ -987,6 +998,10 @@ static int init_server_name(SSL_CONNECTION *s, unsigned int context)
     return 1;
 }
 
+static int init_record_size_limit(SSL_CONNECTION *s, unsigned int context) {
```

Comment:
Since this `init` function is a no-op it seems we can get rid of it entirely (use NULL instead of `init_record_size_limit` in the `ext_defs` array above)